sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -60,55 +60,53 @@
     }
 
     /**
-     * Helper function for calling service with max overloaded parameters that a returns {@link TextAnalyticsPagedFlux}
-     * which is a paged flux that contains {@link AnalyzeSentimentResult}.
+     * Helper function for calling service with max overloaded parameters that returns a {@link Response}
+     * which contains {@link TextAnalyticsResultCollection} of {@link AnalyzeSentimentResult}.
      *
      * @param documents The list of documents to analyze sentiments for.
      * @param options The {@link TextAnalyticsRequestOptions} request options.
      *
-     * @return {@link TextAnalyticsPagedFlux} of {@link AnalyzeSentimentResult}.
+     * @return A mono {@link Response} contains {@link TextAnalyticsResultCollection} of {@link AnalyzeSentimentResult}.
      */
-    TextAnalyticsPagedFlux<AnalyzeSentimentResult> analyzeSentimentBatch(Iterable<TextDocumentInput> documents,
-        TextAnalyticsRequestOptions options) {
+    public Mono<Response<TextAnalyticsResultCollection<AnalyzeSentimentResult>>> analyzeSentimentBatch(
```

Comment:
Name for `TextAnalyticsResultCollection` is being discussed in the API view tool.

Reply:
outdated. This class no longer needed